src/pages/Home/Home.js

```diff
@@ -1,25 +1,49 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
```

Comment:
Two times React is imported

Reply:
No. It's only one time. the red one is from the previous merge. the green one is the updated one.